examples/projects/mongocxx/pkg-config/static/build.sh

```diff
@@ -16,6 +16,14 @@ set -o pipefail
     exit 1
   )
 
+# Sanity-check that static libmongoc is required. Regression test for CXX-3290.
+(pkg-config --print-requires libmongocxx-static | grep -- mongoc2-static || (
+  (
+    echo "Expected mongoc2-static to be required" >&2
+    exit 1
+  )
+))
```

Comment:
```suggestion
if ! pkg-config --print-requires libmongocxx-static | grep -q "mongoc2-static"; then
  echo "Expected mongoc2-static to be required" >&2
  exit 1
fi
```
Lot of subshells; this can be reduced to a straightforward conditional.

Use `-q` to silence grep matches + per grep manpages:

> If no `FILE` is given [...] nonrecursive searches read standard input.

Reply:
Thank you for the suggestion. Updated.